### ISSUE TITLE
Extremely Large Loss

### ISSUE THREAD

**Issue by Songloading** [junyanz/pytorch-CycleGAN-and-pix2pix#993]
Hi, I am now trying to debug my code since I made some minor modifications based on the default settings. When I try to train on the cycle gan model, both D_A and G_A losses go to a really large scale (3000 and -3000) after 10 epochs training. This is really weird since my original training (same dataset) only has D_A less than 1. The generated images (fake_B) have really weird pattern: I can see some edges that could be identified to original real_A, but they all have a really bright color (red, shining green) that are not existing in both domain A and domain B. Do you have any idea on what I am facing? I really appreciate it!

**Comment by junyanz** [comment]
I am not sure -3000 is possible as we are using least square GAN objective (LSGAN) for CycleGAN training. The LSGAN loss is non-negative.  If you modify the loss, you might have some bugs there.

**Comment by Songloading** [comment]
Thanks, I found that I force the model to use wgan.
